packages/react-devtools-shared/src/devtools/views/Components/InspectedElementStyleXPlugin.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import * as React from 'react';
+import KeyValue from './KeyValue';
+import Store from '../../store';
+import sharedStyles from './InspectedElementSharedStyles.css';
+import styles from './InspectedElementStyleXPlugin.css';
+
+import type {InspectedElement} from './types';
+import type {FrontendBridge} from 'react-devtools-shared/src/bridge';
+import type {Element} from 'react-devtools-shared/src/devtools/views/Components/types';
+
+type Props = {|
+  bridge: FrontendBridge,
+  element: Element,
+  inspectedElement: InspectedElement,
+  store: Store,
+|};
+
+export default function InspectedElementStyleXPlugin({
+  bridge,
+  element,
+  inspectedElement,
+  store,
+}: Props) {
+  const styleXPlugin = inspectedElement.plugins.find(
```

Comment:
what is the plugin mechanism enabling here? is it just so we can store this extra data on the inspected element only when that plugin exists?

Reply:
It was me thinking _slightly_ ahead to where we may want to experiment with other "plugins" in the future. TBH maybe it's dumb and we'll end up removing it :)